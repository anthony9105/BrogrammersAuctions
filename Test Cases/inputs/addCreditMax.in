login adminUser1 ^^^^
addcredit
newUser12345
logout