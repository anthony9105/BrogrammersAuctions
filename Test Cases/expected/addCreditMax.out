Login successful
Enter username of account to add credit to
Balance is already at the maximum of 999999999
logout successful